addon/components/bs-navbar.js

```diff
@@ -7,38 +7,40 @@ import layout from '../templates/components/bs-navbar';
 
  ### Usage
 
- Used in combination with the following components:
+ Uses the following components by a contextual reference:
 
- - [Components.NavbarContent](Components.NavbarContent.html)
- - [Components.NavbarToggle](Components.NavbarToggle.html)
- - [Components.NavbarNav](Components.NavbarNav.html)
+ - [Components.NavbarHeader](Components.NavbarHeader.html
+ - [Components.NavbarContent](Components.NavbarContent.html
+ - [Components.NavbarToggle](Components.NavbarToggle.html
+ - [Components.NavbarBrand](Components.NavbarBrand.html
+ - [Components.NavbarNav](Components.NavbarNav.html
  - [Components.NavItem](Components.NavItem.html)
 
  ```hbs
- {{#bs-navbar}}
+ {{#bs-navbar as |navbar|}}
    <div class="navbar-header">
-     {{#bs-navbar-toggle}}
+     {{#navbar.toggle}}
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
-     {{/bs-navbar-toggle}}
+     {{/navbar.toggle}}
      <a class="navbar-brand" href="#">Brand</a>
    </div>
-   {{#bs-navbar-content}}
-     {{#bs-navbar-nav}}
-       {{#bs-nav-item}}{{#link-to "alert"}}Alert{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "button"}}Buttons{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "dropdown"}}Dropdown{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "forms"}}Forms{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "accordion"}}Accordion{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "collapse"}}Collapse{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "modal"}}Modals{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "progress"}}Progress bars{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "navs"}}Navs{{/link-to}}{{/bs-nav-item}}
-       {{#bs-nav-item}}{{#link-to "navbars"}}Navbars{{/link-to}}{{/bs-nav-item}}
-     {{/bs-navbar-nav}}
-   {{/bs-navbar-content}}
+   {{#navbar.content}}
+     {{#navbar.nav}}
+       {{#navbar.nav-item}}{{#link-to "alert"}}Alert{{/link-to}}{{/navbar.nav-item}}
```

Comment:
You have the nav-item yielded by the navbar itself, probably because the nav itself is not yet refactored to use contextual components, right? 

In this case I would then change this to to have the actual nav yield the nav-item, as soon as the nav has been refactored!